### DISCUSSION TITLE
Copy/Add Timecode to exported equilateral MAX files for multiple camera edits

### DISCUSSION THREAD

**Discussion by KyleGW** [gopro/labs#110] [Feature Requests]
I am not sure how to get a hold of the GoPro Player team, maybe someone here could direct this to them. The GoPro MAX camera with labs firmware will keep a precise timecode, and can be used with something like Tentacle Sync E and their Tentacle Timebar phone app which will also generate a QR code for setting a precise synced timecode across multiple cameras. 

Unfortunately, you have to transcode the .360 file to equilateral format to use the footage in an editor like Davinci Resolve or Adobe Premier. When the GoPro player exports the equilateral file, the timecode isn't copied into the export. This make syncying footage more difficult than it needs to be when running multiple cameras. 

I can use ffprobe to see the Timecode metadata on the original .360 file, and go through and manually set this in the new files in the editor, which is a real PITA. I tried using the UDTAcopy tool with the export mov and supplying the original .360 file, like the 12gb chapters page shows to do with a combined .360 file, but that didn't work. I haven't hit on the right FFMPEG or Exiftool command yet either to copy or set the timecode metadata in the exported file either. Any help here would be appreciated.

**Comment by dnewman-gpsw** [comment]
I forwarded the question to the desktop team.  I expect there is a copy command via ffmpeg.

**Reply by KyleGW** [reply]
Thank you!

**Reply by stevegopro** [reply]
time code is not carried over on to rectilinear exports from GoPro studio 2.2
all clips start at zero !

**Comment by jvandalsem-gpfw** [comment]
Hi Kile,  
Timecode retention currently isn't supported on Player for Mac, but it is on Player for Windows.  Are you on Mac or Windows Player currently?   

Timecode retention on Windows had some bugs fixed recently as well, so can you confirm you are on the latest version if you're on windows?   

Thanks.

**Reply by fxstein** [reply]
Really need the Timecode to work on the Mac. Hopefully this gets to the right team. 

Meanwhile anybody has an example of how to do the mentioned ffmpeg copy for the Timecode?

**Reply by stevegopro** [reply]
still ( April 2024) doesn't work on a Mac !   timecode export

**Reply by stevegopro** [reply]
September 2025, a new “professional” max 2 just released and still no timecode on the exported files.

**Reply by dnewman-gpsw** [reply]
I filed the report.  Player is getting more attention with MAX2 so this should be fixed.

**Comment by KyleGW** [comment]
I updated the player to version 2.1.6.3155 and exported a .360 to .mov and the timecode data is now showing. Thank you!

As an aside - where can we find release notes for the latest versions of the player software? I searched support and also used popular search engines and found no mention of release notes or that anything related to timecode was fixed or added. The only article I found related to release notes was the link below, which is out of date. 

https://community.gopro.com/s/article/Software-Update-Release-Information?language=en_US

Thanks again for fixing this, it is immensely helpful and eliminates a time-consuming manual step when working with many video files.

**Reply by stevegopro** [reply]
no it isn't, still no timeday/timecode on cac gopro player max2 equilateral exports. all clips start at 0

**Comment by jjsereday** [comment]
Still having the same issue with the Max2 exports not carrying over timecode - its been a year since its release and still not working can they please fix this?

**Reply by stevegopro** [reply]
not much point in having timecode sync when it isn't carried over to any editing platform !

how difficult can it be ?
